Python/Product/PythonTools/PythonTools/Debugger/CustomDebuggerEventHandler.cs

```diff
@@ -83,5 +84,28 @@ public class CustomDebuggerEventHandler : IVsCustomDebuggerEventHandler110 {
                 "interpreter, please disable PGO.";
             MessageBox.Show(content, "PGO Is Not Supported", MessageBoxButton.OK, MessageBoxImage.Error);
         }
+
+        private void SetDebugOptions(IDebugEngine2 engine) {
+            var pyService = _serviceProvider.GetPythonToolsService();
+
+            var options = new StringBuilder();
+            if (pyService.DebuggerOptions.WaitOnAbnormalExit) {
+                options.Append(";" + AD7Engine.WaitOnAbnormalExitSetting + "=True");
+            }
+            if (pyService.DebuggerOptions.WaitOnNormalExit) {
+                options.Append(";" + AD7Engine.WaitOnNormalExitSetting + "=True");
+            }
+            if (pyService.DebuggerOptions.TeeStandardOutput) {
+                options.Append(";" + AD7Engine.RedirectOutputSetting + "=True");
+            }
+            if (pyService.DebuggerOptions.BreakOnSystemExitZero) {
+                options.Append(";" + AD7Engine.BreakSystemExitZero + "=True");
+            }
+            if (pyService.DebuggerOptions.DebugStdLib) {
+                options.Append(";" + AD7Engine.DebugStdLib + "=True");
+            }
+
+            engine.SetMetric(AD7Engine.DebugOptionsMetric, options.ToString());

```

Comment:
I didn't do it because PythonDebugOptions is internal, but looks like we already have [InternalsVisibleTo] for it, so it's not an issue. I'll make the change.